doc/releases/reef.rst

```diff
@@ -0,0 +1,219 @@
+  be decoded as boolean values in JSON. This also applies to the
+  ``is_truncated`` field returned by ``/subscriptions/<sub-name>``.
+* RGW's response of ``Action=GetTopicAttributes&TopicArn=<topic-arn>`` REST 
+  API now returns ``HasStoredSecret`` and ``Persistent`` as boolean in the JSON
+  string that is encoded in ``Attributes/EndPoint``.
+* All boolean fields that were previously rendered as strings by the 
+  ``rgw-admin`` command when the JSON format was used are now rendered as
+  boolean. If your scripts and tools rely on this behavior, update them
+  accordingly. The impacted field names are:
+  * ``absolute``
+  * ``add``
+  * ``admin``
+  * ``appendable``
+  * ``bucket_key_enabled``
+  * ``delete_marker``
+  * ``exists``
+  * ``has_bucket_info``
+  * ``high_precision_time``
+  * ``index``
+  * ``is_master``
+  * ``is_prefix``
+  * ``is_truncated``
+  * ``linked``
+  * ``log_meta``
+  * ``log_op``
+  * ``pending_removal``
+  * ``read_only``
+  * ``retain_head_object``
+  * ``rule_exist``
+  * ``start_with_full_sync``
+  * ``sync_from_all``
+  * ``syncstopped``
+  * ``system``
+  * ``truncated``
+  * ``user_stats_sync``
+* RGW: The Beast front end's HTTP access log line uses a new
+  ``debug_rgw_access`` configurable. This has the same defaults as
+  ``debug_rgw``, but it can be controlled independently.
+* RGW: The pubsub functionality for storing bucket notifications inside Ceph is
+  removed. Neither it nor the pubsub zone should be used anymore. The following
+  have also been removed: REST operations, radosgw-admin commands for
+  manipulating subscriptions, and commands for fetching and acking the
+  notifications. 
+  
+  If the endpoint to which the notifications are sent is down or disconnected,
+  we recommend that you use persistent notifications to guarantee their
+  delivery. If the system that consumes the notifications has to pull them
+  (instead of the notifications being pushed to the system), use an external
+  message bus (for example, RabbitMQ or Kafka) for that purpose. 
+* RGW: The serialized format of notification and topics has changed. This means
+  that new and updated topics will be unreadable by old RGWs. We recommend
+  completing the RGW upgrades before creating or modifying any notification
+  topics.
+* RGW: Compression is now supported for objects uploaded with Server-Side
+  Encryption. When both compression and encryption are enabled, compression is
+  applied before encryption. Earlier releases of multisite do not replicate
+  such objects correctly, so all zones must upgrade to Reef before enabling the
+  `compress-encrypted` zonegroup feature: see
+  https://docs.ceph.com/en/reef/radosgw/multisite/#zone-features and note the
+  security considerations.
+  
+Unsorted
+~~~~~~~~
+
+* The ``AT_NO_ATTR_SYNC`` macro is deprecated. Use the standard 
+  ``AT_STATX_DONT_SYNC`` macro. The ``AT_NO_ATTR_SYNC`` macro will be removed
+  in the future.
```

Comment:
@zdover23 - Edited/reviewed version from 2023 June 18: The ``AT_NO_ATTR_SYNC`` macro has been deprecated in favor of the standard ``AT_STATX_DONT_SYNC`` macro. The ``AT_NO_ATTR_SYNC`` macro will be removed in the future.

(This seems important for consistency with the following bullet point.)

Reply:
Accepted.

Reply:
This should be moved to CephFS.